docs/overview/component/filter-gateway.md

```diff
@@ -1,22 +1,72 @@
 # Vald Filter Gateway
 
-Vald Filter Gateway is the component to communicate user-defined ingress / egress filter components with requests.
-
-This page introduces the overview of Vald Filter Gateway.
+Vald Filter Gateway is the component to communicate user-defined ingress/egress filter components with requests.
 
 ## Responsibility
 
-Vald Filter Gateway is the optional component in the Vald cluster.
+Vald Filter Gateway is the optional component in the Vald cluster implemented for available user-defined filtering.
 
-It is implemented for available of user-defined filtering.
-The responsibility is bypass between the Vald cluster and user-defined ingress filter (or egress filter) components.
+The responsibility is bypassed between the Vald cluster and the user-defined ingress filter (or egress filter) component.
 
 ## Features
 
 ### Ingress Filtering
 
-<!-- TODO: describe about ingress -->
+Ingress filtering is the pre-process before the request is processed by the Vald LB Gateway.
+
+This is useful, e.g., when you want to vectorize object data (when you want to handle object data directly as the request query) or when you want to pad the vector, etc.
+
+Here are the steps of pre-processing with the ingress filter:
+1. The user's request flows from the filter gateway to the ingress filter component when using the Ingress filter.
+1. Ingress filter performs arbitrary processing for each user and returns the result to the filter gateway.
+1. The filter gateway passes the ingress filter result to the Vald LB Gateway.(The Vald LB Gateway migration process is the same as when the ingress filter is not used. Click [here](../../overview/data-flow.md) for a detailed data flow.)
+
+<img src="../../../assets/docs/overview/component/filter-gateway/ingress_filtering.svg" alt="example data flow with ingress filtering" />
+
+There are two types of ingress filtering methods: generate vector and vector filtering.
+In generate vector, for example, you can vectorize object data using your favorite model.
+The filtering vector allows you to filter out vectors that contain outliers so that they are not indexed.
```

Comment:
```suggestion
There are two types of ingress filtering methods: generate vector and vector filtering.
In generating vector, for example, you can vectorize object data using your favorite model.
The filtering vector allows you to filter out vectors that contain outliers so that they are not indexed.
```